### ISSUE TITLE
[Doc]: Redundant layout in Matplotlib documentation

### ISSUE THREAD

**Issue by codewithdev** [matplotlib/matplotlib#26495]
### Documentation Link

https://matplotlib.org/stable/api/axes_api.html#projection

### Problem

* All the content in the table of content already contained in the "On this page"  .  We can probably remove the Table of Content from the top. It makes no sense adding. Users usually click the pages from the right-most On this page expander.

**Comment by akshayamadhuri** [comment]
@codewithdev  i would like to work on this issue

**Comment by codewithdev** [comment]
I would happy to collaborate.

**Comment by jklymak** [comment]
This is a _very_ long API doc, and having an intro TOC with nested levels helps get to the correct place quickly.  Also on mobile you don't see the "On this page".  I think the duplication is warranted and helpful for this page.

**Comment by codewithdev** [comment]
cool @jklymak would love to work on this.

**Comment by codewithdev** [comment]
Seems like the issue is with the Sphinx theme that mpl is currently using.

**Comment by timhoffm** [comment]
No activity, and the tendency goes towards keeping as is https://github.com/matplotlib/matplotlib/issues/26495#issuecomment-1675246532

**Comment by story645** [comment]
Yeah I'm still not a fan of the ToC but I also figure if we were to get rid of it, it'd be in the context of some larger scale reorg so I agree we should closing this specific issue since I don't think it's actionable.
